packages/core/src/App/Containers/RealAccountSignup/terms-of-use-form.js

```diff
@@ -1,4 +1,4 @@
-import { isDesktop } from '@deriv/shared/utils/os';
+import { isDesktop } from '@deriv/shared';
```

Comment:
`isDesktop` is a screen size import. Please use `isDesktopOs`.

Reply:
@reza-fs mentioned that we can switch to using the screen import instead of the OS check here, so we will pass this through.